src/AppBundle/Entity/Card.php

```diff
@@ -2614,6 +2621,30 @@ public function getSchemeAcceleration()
         return $this->schemeAcceleration;
     }
 
+	/**
```

Comment:
Looks like a tab here when the rest of the file is spaces so this looks misaligned.